astropy/utils/data_info.py

```diff
@@ -0,0 +1,296 @@
+import os
```

Comment:
Maybe note somewhere in this file that it is extensively tested as part of `table.tests`, not in `utils.tests`?